pyclam/criterion.py

```diff
@@ -111,6 +111,28 @@ def __call__(self, cluster: Cluster) -> bool:
         return distance > 0.25
 
 
+class Clause(ClusterCriterion):
```

Comment:
I would say `SelectionClause` may make this a bit clearer

Reply:
removed this decision-tree based criterion